### PR TITLE
Team4 Agent report

### PR DESCRIPTION
**Pull request by MikeIsMike**

### CODE REVIEW

File: 03_gamedesign/index.tex [hunk 1]
```diff
@@ -209,6 +209,20 @@ \subsubsection{Code testing}
 
 While Go is strongly typed and would mean that most errors can be detected at compile-time (or even at time of coding with its performant language server), having unit and integration tests greatly improved the maintainability and ease of development of the project--these issues were particularly important as this was a large-scale group software engineering project--developers stepping on each other's toes is a common occurrence in non-tested group project code. Henceforth, tests were required for non-trivial server-side code.
 
+<<<<<<< HEAD
```
Comment (NGovani) [inline 03_gamedesign/index.tex]: this needs to be removed

File: 03_gamedesign/index.tex [hunk 2]
```diff
@@ -209,6 +209,20 @@ \subsubsection{Code testing}
...
+
+\subsection{Continuous Development}
+
+On receiving PR approval, passing automated tests and finally merging into the main codebase, the visualisation website is automatically rebuilt with the latest changes. This saved time as manual builds were not required. The builds were produced on a reference system (identical to that mentioned in~\ref{sec:GD:implementation:practices}), ensuring consistent builds free from system-specific quirks. Further, automated builds meant that the website always runs on the latest codebase.
+=======
```
Comment (NGovani) [inline 03_gamedesign/index.tex]: so does this

File: 03_gamedesign/index.tex [hunk 3]
```diff
@@ -219,4 +233,5 @@ \subsubsection{Continuous Integration}
 
 \subsubsection{Continuous Deployment}
 
-On receiving PR approval, passing automated tests and finally merging into the main codebase, the visualisation website is automatically rebuilt with the latest changes. This saved time as manual builds were not required. The builds were produced on a reference system (identical to that mentioned in the Continuous Integration section), ensuring consistent builds free from system-specific quirks. Further, automated builds meant that the website always runs on the latest codebase.
\ No newline at end of file
+On receiving PR approval, passing automated tests and finally merging into the main codebase, the visualisation website is automatically rebuilt with the latest changes. This saved time as manual builds were not required. The builds were produced on a reference system (identical to that mentioned in the Continuous Integration section), ensuring consistent builds free from system-specific quirks. Further, automated builds meant that the website always runs on the latest codebase.
+>>>>>>> 962869f9957a17b0764e8a570c39fa55a16b5576
```
Comment (NGovani) [inline 03_gamedesign/index.tex]: remove

**Review by NGovani — CHANGES_REQUESTED** [review]

File: 03_gamedesign/index.tex (hunk 1 above)
Comment (yp717) [reply 03_gamedesign/index.tex]: there shouldn't be any edits in gamedesign from team4 as far as I know - is this a mistake?

**Review by yp717 — DISMISSED** [review]
remove changes from file not related to PR

**Comment by ezgimez** [comment]
this is a weird and very confusing PR

**Comment by ezgimez** [comment]
you are strongly advised NOT TO edit any submitted sections in the previous sections in the report.

**Comment by ezgimez** [comment]
do git pull, open a new branch, make a new PR. make some effort plz :)

File: 03_gamedesign/index.tex (hunk 1 above)
Comment (akowal3) [reply 03_gamedesign/index.tex]: fixing now

Comment (tobhil98) [reply 03_gamedesign/index.tex]: Done

File: 03_gamedesign/index.tex (hunk 2 above)
Comment (tobhil98) [reply 03_gamedesign/index.tex]: Done

File: 03_gamedesign/index.tex (hunk 3 above)
Comment (tobhil98) [reply 03_gamedesign/index.tex]: Done

**Review by NGovani — APPROVED** [review]
It's not breaking and everything looks good in pdf form. So good to go

**Pull request merged**